polkadot/runtime/westend/src/xcm_config.rs

```diff
@@ -172,8 +173,8 @@ pub type Barrier = TrailingSetTopicAsId<(
 			AllowTopLevelPaidExecutionFrom<Everything>,
 			// Subscriptions for version tracking are OK.
 			AllowSubscriptionsFrom<OnlyParachains>,
-			// Collectives and Fellows plurality get free execution.
-			AllowExplicitUnpaidExecutionFrom<CollectivesOrFellows>,
+			// Messages coming from system parachains need not pay for execution.
+			AllowExplicitUnpaidExecutionFrom<IsChildSystemParachain<ParaId>>,
```

Comment:
does this change not take away unpaid execution from `Fellows`?

Reply:
looks like so, [IsChildSystemParachain matches only on exact parachain origin and not its children](https://github.com/paritytech/polkadot-sdk/blob/8428f678fe5179399f6854e194f1c8b30a9102f9/polkadot/xcm/xcm-builder/src/barriers.rs#L317C3-L321C4);

whereas [Fellows is a child location of Collectives system chain](https://github.com/paritytech/polkadot-sdk/blob/d55d4f64b50c858bce6fa1a429f50d486cc67d49/polkadot/runtime/westend/src/xcm_config.rs#L151).